Tests/AuthTests/CacaoSignerTests.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+import XCTest
+@testable import Auth
+import TestingUtils
+
+class CacaoSignerTest: XCTestCase {
+
+    let payload = CacaoPayload(
+        iss: "did:pkh:eip155:1:0x22Fe071b3631f155F0d8f4c9377D3309cB904E10",
```

Comment:
Maybe we could create a DIDPKH object co we can easily derive caip-10 account or an address from it?

Reply:
Check SignerAddress please. There is `static func from(iss: String) -> String`
Looks good for you?

Reply:
I was thinking more about something like:
 ```
let signerDid = DIDPKH(account: Account)
signerDid.account ...
```
but if it works it works.